.github/workflows/nightly-smoke-tests.yml

```
@@ -0,0 +1,28 @@
+name: Nightly Smoke Tests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  smoke_tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Checkout code
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.19'
```

```suggestion
          go-version: 'stable'
```

And I just realized that TF repo's smoke test workflow file may also need this...